backend/app/tests/api/routes/test_creds.py

```
@@ -394,6 +392,7 @@
     # Create OpenAI credentials
     openai_credential = {
         "organization_id": org.id,
+        "project_id": project.id,
```

Ruff (0.12.2)

Undefined name `project`

(F821)